WalletWasabi.Backend/Config.cs

```diff
@@ -24,6 +25,54 @@ public class Config : IConfig
 		[JsonProperty(PropertyName = "BitcoinRpcConnectionString")]
 		public string BitcoinRpcConnectionString { get; private set; }
 
+		[JsonProperty(PropertyName = "MainNetBitcoinCoreHost")]
+		public string MainNetBitcoinCoreHost { get; internal set; }
+
+		[JsonProperty(PropertyName = "TestNetBitcoinCoreHost")]
+		public string TestNetBitcoinCoreHost { get; internal set; }
+
+		[JsonProperty(PropertyName = "RegTestBitcoinCoreHost")]
+		public string RegTestBitcoinCoreHost { get; internal set; }
+
+		[JsonProperty(PropertyName = "MainNetBitcoinCorePort")]
+		public int? MainNetBitcoinCorePort { get; internal set; }
+
+		[JsonProperty(PropertyName = "TestNetBitcoinCorePort")]
+		public int? TestNetBitcoinCorePort { get; internal set; }
+
+		[JsonProperty(PropertyName = "RegTestBitcoinCorePort")]
+		public int? RegTestBitcoinCorePort { get; internal set; }
```

Comment:
You are doing lot's of copy pasta because you don't use a custom `JsonConverter` for `IPEndpoint`.
Actually it should not even be `IPEndpoint` by `Endpoint` as it could be a `DnsEndpoint`.

Reply:
It was done this way to be consistent with the client side code. Which cannot be touched, because people already have existing settings and this would ruin that.  

About IPEndPoint, your Node.Connect eats IPEndPoint, not DnsEndPoint, there may be other things, but I'd be up for changing, however at this point I'm not sure what's the difference, I never used DnsEndPoint and I couldn't really grasp the concept with 10min reading about it, so I won't bother until it's needed.

Reply:
> About IPEndPoint, your Node.Connect eats IPEndPoint, not DnsEndPoint

Yeah I really need to fix that... If I manage to fix that I can then accept `TorEndpoint` or stuff like this.

The definition of an endpoint is just "specification on where to connect".
A `DnsEndpoint` need to be resolve when one is connecting.
A `TorEndpoint` (if it existsed) need to be using a socks proxy.
A `IPEndpoint` is what you know.